depthai_sdk/docs/source/blob_nnet_manager.rst

```diff
@@ -0,0 +1,76 @@
+======================
+Blob and Nnet managers
+======================
+
+Blob manager
+------------
+
+``Blob manager`` is a class that is made to help you with downloading neural networks as MyriadX blobs.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+``Blob manager`` is very easy and straight forward to use. We declare it and pass which project we want to use as it's argument.
+
+.. literalinclude:: ./examples/code_fractions/blob.py
+   :language: python
+   :linenos:
+
+After that, the ``blob`` is stored in our variable and we can then pass it to our ``Nnet manager``, as we will se bellow.
+Of course the ``face-detection-retail-0004`` is not the only project that we can use. All other projects that are supported are:
+
+.. include::  blob_projects.rst
+
+Nnet manager
+------------
+
+``Blob manager`` is a class that is made to help you with setting up neural networks.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+As the ``Blob manager`` and ``Nnet manager`` work together, we will need both to use our neural network. We will also need to use the ``Pipeline manager`` and the ``Preview manager`` to make everything work together.
```

Comment:
This is false, as BlobManager and NNetManager can (and often are) used separately. 
BlobManager is useful if you want to have the ability to download models based on model config or custom model zoo.
Nnet Manager only requires blob path, which can be provided both from BlobManager but also directly from `blobconverter` module.

I often do `blobPath=blobconverter.from_zoo(...)` as it's simpler

Reply:
Regarding PipelineManager and PreviewManager connections:
- PipelineManager is able to integrate with `NNetManager`, as it's needed to both add neural network node to the pipeline (`addNN` method) and we need to sync OpenVINO version between nn and pipeline (hence `setNnManager` method). This is one way sync and is mostly optional
- NNetManager is able to integrate with PreviewManager for convenience, but it's also able to parse raw frames passed to it.

With the comments above, my point is that we should show our users that these manager/helper classes are flexible and modular, allowing to pick only the ones you find most suitable